runs/updateConfig.sh

```diff
@@ -681,19 +681,17 @@ updateConfig(){
 	fi
 	if ! grep -Fq "soc_vag_intervall=" $ConfigFile; then
 		echo "soc_vag_intervall=60" >> $ConfigFile
+		echo "soc_vag_intervallladen=10" >> $ConfigFile
```

Comment:
Die Änderung bitte wieder entfernen. Das hatte schon vorher seinen Sinn. ;-)

Reply:
Vorher wurde jede Variable separat überprüft, so hätte ich nur die erste geprüft und alle zur conf. ergänzt. So hast Du es bei Tronigy doch auch gemacht. Wo ist der Unterschied?

Reply:
Der Unterschied ist, dass das zweite Intervall erst später ergänzt wurde. Es gibt also potentiell Installationen, bei denen `soc_vag_intervall` vorhanden ist, `soc_vag_intervallladen` jedoch nicht. Für diesen Fall müssen beide Einträge separat geprüft werden.

Reply:
Das VAG Modul habe ich ja in einem Rutsch gemacht, es dürfte bei jedem Nightly-Nutzer immer stets beides hinzugefügt worden sein. Daher erkenne ich das Problem hier in diesem Fall nicht, aber nehme es natürlich auch gern wieder raus. Ich richte mich da nach Dir

Reply:
Stimmt, hatte mich mit einem anderen Modul vertan. Kann also so bleiben.